features/step_definitions/registry.rb

```diff
@@ -375,21 +375,24 @@
 # Generate default route
 Given /^I enable image-registry default route$/ do
   ensure_admin_tagged
-#  ensure_destructive_tagged
```

Comment:
Changing registry should be destructive.

Reply:
@liangxia just enable route, it doesn't infect others.

Reply:
@liangxia if we add `ensure_destructive_tagged`, then we need add destructive label for all cases which need this step. But some ones are actually not destructive cases